public/stylesheets/components/forms/_radio-buttons.scss

```diff
@@ -70,6 +70,40 @@
   background-color: $radio-fill-color;
 }
 
+//add traffic light colors
```

Comment:
Can you add a screenshot? So it's easier to review without having to pull and run it locally

Reply:
<img width="332" alt="Screen Shot 2020-03-16 at 10 42 58 PM" src="https://user-images.githubusercontent.com/42791107/76816900-d22cc380-67d7-11ea-857b-ba148fc4e166.png">
<img width="344" alt="Screen Shot 2020-03-16 at 10 42 52 PM" src="https://user-images.githubusercontent.com/42791107/76816902-d2c55a00-67d7-11ea-9f97-56f7c0517bc7.png">
<img width="335" alt="Screen Shot 2020-03-16 at 10 42 45 PM" src="https://user-images.githubusercontent.com/42791107/76816903-d35df080-67d7-11ea-91b5-11b7a6650dc5.png">
<img width="327" alt="Screen Shot 2020-03-16 at 10 42 39 PM" src="https://user-images.githubusercontent.com/42791107/76816905-d3f68700-67d7-11ea-9f71-ad58e0c8a5a8.png">

Reply:
(Just add it in the description/notes of the PR next time!)